alembic/versions/2136a1f22f1f_cull_removed_rules.py

```diff
@@ -0,0 +1,47 @@
+"""Cull removed rules
+
+Revision ID: 2136a1f22f1f
+Revises: 2ea9623b21fa
+Create Date: 2015-01-08 12:23:51.829172
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '2136a1f22f1f'
+down_revision = '2ea9623b21fa'
+
+from alembic import op
+import sqlalchemy as sa
+
+import fmn.lib.models
+
+
+def upgrade():
+    engine = op.get_bind().engine
+    session = sa.orm.scoped_session(sa.orm.sessionmaker(bind=engine))
+
+    # Find all rules that got removed in this PR and nuke them
+    # https://github.com/fedora-infra/fmn.rules/pull/21
+    goners = [
+        'fmn.rules:pkgdb_acl_user_remove',
+        'fmn.rules:pkgdb_branch_clone',
+        'fmn.rules:pkgdb_critpath_update',
```

Comment:
This one was moved, not removed: https://github.com/fedora-infra/fmn.rules/pull/21/files#diff-f1b769f34e51b7aa715c3fda9ab9f599R123